README.md

```diff
@@ -1,35 +1,142 @@
 ![](https://img.shields.io/badge/Certification%20Level-Community-28A745?link=https://github.com/cyberark/community/blob/master/Conjur/conventions/certification-levels.md)
 
-## CyberArk Ansible Conjur Collection
+# CyberArk Ansible Conjur Collection
 
-****
+This collection contains components to be used with CyberArk Conjur & DAP (Dynamic Access Provider)
+hosted in [Ansible Galaxy](https://galaxy.ansible.com/cyberark/conjur).
 
-### cyberark.conjur
+## Table of Contents
+- [CyberArk Ansible Conjur Collection](#cyberark-ansible-conjur-collection)
+  * [Requirements](#requirements)
+  * [Conjur Ansible Role](#conjur-ansible-role)
+    + [Usage](#usage)
+    + [Role Variables](#role-variables)
+    + [Dependencies](#dependencies)
+    + [Example Playbook](#example-playbook)
+    + [Summon & Service Managers](#summon---service-managers)
+    + [Recommendations](#recommendations)
+  * [conjur_variable Lookup Plugin](#conjur_variable-lookup-plugin)
```

Comment:
This needs to be updated to match the updates below - maybe regenerate the TOC?

I'm always a fan of dropping the header from the TOC, bc it artificially indents everything one level. that'd be worth doing too imo.

Reply:
I actually was just working on this before you posted this review lol